entrypoint.sh

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env bash
 
 # Generated environment for CRON jobs
-/usr/bin/printenv | sed 's/^\(.*\)$/export \1/g' | grep -E "^export TZ|RECEIVE_IMAP" > /var/local/redmine/scripts/env_receive_imap.sh
```

Comment:
This line has been removed, but it's probably needed for the fist line of the crontab (see https://github.com/eaudeweb/redmine.docker/blob/master/crontab#L1)